Snowflake.Data/Core/SFStatement.cs

```diff
@@ -60,6 +60,9 @@ class SFStatement
 
         string _bindStage = null;
 
+        // the query id of the last succeeded query

```

Comment:
its also for unsucceeded query? like executed, parsed but failed regarding syntax or inability to ie. update db, right?

Reply:
fixed